habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hl_neural.yaml

```diff
@@ -0,0 +1,29 @@
+name: "HierarchicalPolicy"
+obs_transforms:
+  add_virtual_keys:
+    virtual_keys:
+      "goal_to_agent_gps_compass": 2
+hierarchical_policy:
+  high_level_policy:
+    name: "NeuralHighLevelPolicy"
+    allow_other_place: False
+    hidden_dim: 512
+    use_rnn: True
+    rnn_type: 'LSTM'
+    backbone: resnet18
+    normalize_visual_inputs: False
+    num_rnn_layers: 2
+    policy_input_keys:
+      - "robot_head_depth"
+  use_skills:
```

Comment:
I don't like the habit of defining all options in the configs and then having a list that defines a subset of those options. I would prefer `use_skills` to define the whole thing instead of bing a key.

Reply:
(TLDR:  I think there should only be defined_skills and not use_skills)

Reply:
Agreed. This is now fixed. Everything is now under defined_skills.